x-pack/plugins/security_solution/server/endpoint/services/artifacts/manifest_manager/manifest_manager.ts

```diff
@@ -433,28 +416,58 @@ export class ManifestManager {
       return errors;
     }
 
+    this.logger.debug(`Creating [${artifactsToCreate.length}] artifacts`);
+
     const { artifacts: fleetArtifacts, errors: createErrors } =
       await this.artifactClient.bulkCreateArtifacts(artifactsToCreate);
 
+    this.logger.info(`Count of artifacts created: ${fleetArtifacts?.length ?? 0}`);
```

Comment:
Should this be also at debug level? Or only show it if there are new artifacts created?

Reply:
I can go either way on this one. I just thought it would be good for the task to report what it did